Description: Create setup.py in sources
 This patch create a quick setup.py in sources.
Author: Stephane Neveu
Last-Update: 2017-06-16
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+
+#Authors:
+#Stephane Neveu <stefneveu@gmail.com>
+
+import os
+import os.path
+from distutils.core import setup
+
+setup(
+    name="brutespray",
+    author="Shane Young and Jacob Robles",
+    description="BruteSpray takes nmap GNMAP/XML output and automatically brute-forces services with default credentials using Medusa.",
+    license="MIT/X11",
+    url="https://github.com/x90skysn3k/brutespray",
+    scripts=[
+    ("brutespray.py")
+    ],
+)
+
